CHANGES.txt

```diff
@@ -21,6 +21,10 @@ Changes
 Fixes
 =====
 
+ - Use "crate" superuser as user session if a system query is executed. The
+   stats tables ``sys.jobs`` and ``sys.jobs_log`` list this user to prevent of
+   showing NULL instead.
```

Comment:
i would leave the stats stuff out, it has nothing to do with it. what about: "Internal system queries are now executed under the 'crate' superuser if user management is enabled.